### DISCUSSION TITLE
I'd like you to assist me in getting in touch with the author to the mod ProjectRed so that i can get permission to use the frame mover idea

### DISCUSSION THREAD

**Discussion by fuzzycomponent** [MrTJP/ProjectRed#1863] [Q&A]
I aspire to develop a game but one of the ideas i plan to implement is not mine. Frames and frame motors are considered intelectual property of the original author of ProjectRed and i am obligated to request royalty free use of only the part i plan to implement in my block voxel game: Frame machines. If the rights to the idea were passed on and enable you to give me permission to directly inspire the frame like devise, and potentially expound on the idea my self i will be able to do what i intended to out of nostalgia for my old minecraft experiences with that mod, and people can enjoy my version of it too (i mean, i brainstorm a lot. i want frame miners to be possible though at the very least).

**Comment by MrTJP** [comment]
Hi. I am the original author of Project Red. But I was certainly not the first with a similar Frame system. Like most things in PR, they were based on RedPower, which to my knowledge was the first with this kind of system. 

You cannot use original code or assets from RedPower. But nothing can stop your from rewriting it from scratch. 

You cannot claim rights to ideas, only implementations of ideas :)
